blog/tooltip-w-teorii-i-praktyce/index.md

```diff
@@ -0,0 +1,168 @@
+
+Istnieje wiele powodów, dla których warto używać toolipów w aplikacji, oto
+niektóre z nich:
+
+- zajmują niewielką ilość miejsca na ekranie, więc nie są uciążliwą formą
+  przesyłania wiadomości w aplikacji.
+- poprawiają użyteczność i ułatwiają wykonywanie określonych zadań.
+- zapewniają użytkownikom informacje na żądanie w momencie, gdy są potrzebne,
+  dzięki czemu użytkownicy mogą podejmować decyzje, skuteczniej poruszać się po
+  interfejsach i rozumieć funkcjonalność elementów interaktywnych.
+- wyświetlają się tylko kontekstowo, dostarczając dodatkowych wyjaśnień i
+  wskazówek. Swoją drogą, jeśli byłyby wyświetlane na stałe, zaśmiecałyby
+  interfejs użytkownika 😉.
+- mogą być nieocenionym wsparciem dla nowych użytkowników, ułatwiając im
+  zrozumienie funkcji, ich poszczególnych elementów i umożliwiając im interakcję
+  z treściami internetowymi. A jak? Na przykład poprzez przypomnienie, co robi
+  dana ikona lub gdy jakaś funkcja jest trudna do wyjaśnienia są pomocne w
+  nawigowaniu.
+- można je stosować zarówno do prezentacji produktów, wdrażania funkcji, czy też
+  szkolenia użytkowników.
+
+## Gdzie warto je stosować?
+
+Wykorzystanie tooltipów może być dość wszechstronne w różnych biznesach, ale
+skupmy się na kilku typowych w branży technologicznej. Można do nich zaliczyć
+między innymi:
+
+- wycieczki wdrożeniowe – gdy użytkownik po raz pierwszy loguje się do
+  aplikacji, może odbyć wycieczkę po aplikacji lub jej funkcjach. Wyświetlane w
+  aplikacji podpowiedzi pokazują każdy aspekt danej funkcji i dostarczają
+  informacji o tym, co robi dana ikona. Wariantem wycieczek mogą być
+  interaktywne przewodniki, czyli podpowiedzi pojawiają się dopiero po
+  interakcji użytkownika z poprzednią podpowiedzią, dzięki czemu są bardziej
+  angażujące niż ogólna wycieczka po aplikacji.
+- wprowadzanie nowych funkcji – z których użytkownicy jeszcze nie korzystali -
+  za pomocą podpowiedzi możesz wyróżnić funkcje, które nie były jeszcze używane
+  i nauczyć użytkowników, jak z nich korzystać
+- zapowiadanie nowych funkcji i zmian w produktach – podpowiedzi doskonale
+  nadają się do ogłaszania zmian w aplikacji czy dodawania nowych funkcji.
+- zapewnienie pomocy, gdy użytkownik utknie w martwym punkcie – jeśli użytkownik
+  nie może ruszyć dalej, tooltipy mogą pomoc w wykonaniu konkretnych czynności
+  lub skierować użytkownika do źródeł „samopomocy”. Jeśli użytkownicy potrzebują
+  więcej informacji, wówczas mogą skorzystać z podpowiedzi zamiast kontaktować
+  się z pomocą techniczną.
+
+## Co jest ważne podczas tworzenia tooltipów?
+
+Chociaż tooltipy nie są nowością w sieci, często są one niewłaściwie stosowane
+lub zaimplementowane. Na co zatem warto zwrócić uwagę przy ich tworzeniu?
+
+- dodawajcie krótki tekst – tooltipy z oczywistym lub zbędnym tekstem nie
+  przynoszą korzyści użytkownikom. Jeśli nie możecie wymyślić pomocnej treści,
+  nie oferujcie podpowiedzi. Dodana podpowiedź powinna być samowystarczalna,
```

Comment:
nie wiem czy "samowystarczalna" to wlasciwe slowo  w tym konteksie. Rozumiem idee, cos jak "self contained" po angielsku. Mozesz je usnuac i zostawic : "Dana podpowiedz powinna byc zwiezle opisana i nie blokowac powiazanych tresci".